.github/ISSUE_TEMPLATE/releases/vs-tools-release.md

```diff
@@ -0,0 +1,30 @@
+# VS Tools Release
+
+VS Version: &lt;version&gt;
+
+## Tasks
```

Comment:
Is it worthwhile calling out that VS releases should be combined with the PT release process when they happen on the same day?

Reply:
Fixed